dump/mapping/bufr_ahicsr.yaml

```diff
@@ -0,0 +1,247 @@
+# (C) Copyright 2025 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache License Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0.
+
+bufr:
+  subsets:
+    - NC021044
+
+  variables:
+    # MetaData
+    timestamp:
+      datetime:
+        year: "*/YEAR"
+        month: "*/MNTH"
+        day: "*/DAYS"
+        hour: "*/HOUR"
+        minute: "*/MINU"
+        second: "*/SECO"
+
+    dataReceiptTime:
+      datetime:
+        year: "*/RCYR"
+        month: "*/RCMO"
+        day: "*/RCDY"
+        hour: "*/RCHR"
+        minute: "*/RCMI"
+
+    receiptTimeSignificance:
+      query: "*/RCTS"
+
+    latitude:
+      query: "*/CLATH"
+
+    longitude:
+      query: "*/CLONH"
+
+    satelliteId:
+      query: "*/SAID"
+
+    satelliteZenithAngle:
+      query: "*/SAZA"
+
+    solarZenithAngle:
+      query: "*/SOZA"
+
+    numberPixelHorizontal:
+      query: "*/NPPR"
+
+    numberPixelVertical:
+      query: "*/NPPC"
+
+    landOrSeaQualifier:
+      query: "*/LSQL"
+
+    geopotentialHeight:
+      query: "*/HITE"
+
+    reportIndicator:
+      query: "*/CORN"
+
+    sensorCentralFrequency:
+      query: "*/RPSEQ11{1}/SCCF"
+
+    sensorBandWidth:
+      query: "*/RPSEQ11{1}/SCBW"
+
+    satelliteInstrumentData:
+      query: "*/RPSEQ11{1}/SIDP"
+
+    radianceType:
+      query: "*/RPSEQ11{1}/RDTP"
+
+    radianceComputationalMethod:
+      query: "*/RPSEQ11{1}/RDCM"
+
+    # ObsValue   
+    cloudType:
+      query: "*/RPSEQ11{1}/CLTP"
+
+    cloudAmount:
+      query: "*/RPSEQ11{1}/CLDMNT"
+
+    cloudFree:
+      query: "*/RPSEQ11{1}/NCLDMNT"
+
+    brightnessTemperature:
+      query: "*/RPSEQ11{1}/TMBRST"
+
+    clearSkyStandardDeviation:
+      query: "*/RPSEQ11{1}/SDTB"
+
+  splits:
+    satId:
+      category:
+        variable: satelliteId
+        map:
+          _173: h8 
+          _174: h9
+
+encoder:
+
```

Comment:
You need to include `dimension` section 

Here is the example from ATMS:
@PraveenKumar-NOAA 
The channel dimension is missing in the IODA file (output)
```
netcdf gdas.t00z.ahi_h9.tm00 {
dimensions:
        Location = 317847 ;
variables:
        int Location(Location) ;
                Location:_FillValue = 2147483647 ;

```

This is because your YAML file does not include the dimension section in the encoder section

Here is the example from ATMS:
https://github.com/NOAA-EMC/spoc/blob/369e647dcad24067f365b5c75b83bc74115abb58/dump/mapping/bufr_atms.yaml#L75-L80

Reply:
@emilyhcliu thanks, done.